tls/s2n_connection.c

```diff
@@ -1033,11 +1033,13 @@ const char *s2n_connection_get_curve(struct s2n_connection *conn)
 {
     PTR_ENSURE_REF(conn);
 
-    if (!conn->secure.server_ecc_evp_params.negotiated_curve) {
-        return "NONE";
+    if (conn->secure.server_ecc_evp_params.negotiated_curve && 
```

Comment:
How does this behave for TLS1.3 when an ECC group is negotiated?

Reply:
Added logic to handle TLS1.3 case where we always return a curve.